### PR TITLE
Add relativePath option

### LINKED ISSUE #15: code to have the json paths relative to a destination folder
Here's the code to have the json paths relative to a destination folder:

	var dest = "public/";

	// Remove dest from manifest
	var bust_formatter = function(json){
	    var new_json = {};
	    for(var key in json) {
	        new_key = key.replace(dest, '');
	        new_json[new_key] = json[key];
	    }
	    return JSON.stringify(new_json);
	};

Use like this

	.pipe(bust({formatter:bust_formatter}))

### PR DESCRIPTION
**Pull request by benface**
Closes #15

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,13 +13,15 @@ var DEFAULT_OPTIONS = {
 	length: 0,
 	transform: Object,
 	formatter: JSON.stringify,
+	relativePath: '/',
```
Comment (UltCombo) [inline index.js]: I guess `'.'` might fit better here, I usually don't like (ab)using multiple consecutive slashes in paths.

File: index.js [hunk 2]
```diff
@@ -75,7 +77,7 @@ module.exports = exports = function(options) {
 		hashingPromises.push(
 			Promise.try(hash.bind(undefined, file, options)).then(function(hashed) {
 				if (typeof hashed !== 'string') throw error('Return/fulfill value of `options.algo` must be a string');
-				hashes[relativePath(file.cwd, file.path)] = sliceHash(hashed, options);
+				hashes[relativePath(file.cwd + '/' + options.relativePath, file.path)] = sliceHash(hashed, options);
```
Comment (UltCombo) [inline index.js]: With the suggestion [above](https://github.com/UltCombo/gulp-buster/pull/22/files#r74712092), you should be able to do `relativePath(file.cwd, options.relativePath, file.path)`

**Comment by UltCombo** [comment]
Nice work!

I've added a few review comments above.

It would be nice to have an unit test for this option as well. I usually don't mind writing the tests myself, but I'm on a tight schedule right now and can't spend time on this right now.

**Comment by benoitr007** [comment]
@UltCombo Thanks! Please check again when you can. :)

**Comment by UltCombo** [comment]
Awesome work!

I'll make a few adjustments and push it right away.

**Comment by UltCombo** [comment]
I've squashed your commits and made small adjustments to the `relativePath` function and readme: d7336b7d63ad9a8a753e6f07033b83d8f5aab111. Thanks!

Published `gulp-buster@1.1.0`! 🎉

**Comment by benface** [comment]
👍

**Pull request closed without merging**